cms_form/models/cms_form_mixin.py

```diff
@@ -214,10 +216,13 @@ def form_model(self):
         # queue_job tries to read properties. Be defensive.
         return self.env.get(self._form_model)
 
-    def form_fields(self):
+    def form_fields(self, hidden=None):
```

Comment:
It took me quite a while to understand the use of the parameter and its effect. Still feels quite weird to me to get 3 different results using True, False or None for what looks like a boolean field. IMO a docstring is needed in such case.

Reply:
right, forgot the docstring!

Reply:
added